lib/twilio-ruby/twiml/twiml.rb

```diff
@@ -37,6 +35,7 @@ def to_s(xml_declaration = true)
         return ('<?xml version="1.0" encoding="UTF-8"?>' + xml) if xml_declaration
         xml
       end
+      alias to_xml to_s
```

Comment:
If the `alias` has to go in a specific order to work properly, then I think it makes more sense to just explicitly define `to_xml` and call `to_s` within.

Also, does this error not exist for:
```
lib/twilio-ruby/jwt/client_capability.rb
15:      alias to_s to_jwt
```